File: src/BreakfastProvider.Api/Models/Responses/WaffleResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record WaffleResponse
{
    public Guid BatchId { get; init; }
    public List<string> Ingredients { get; init; } = [];
    public List<string> Toppings { get; init; } = [];
    public DateTime CreatedAt { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/WaffleResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

/** Twin of C# {@code WaffleResponse}. */
public record WaffleResponse(UUID batchId, List<String> ingredients, List<String> toppings, Instant createdAt) {
}
